helpers/config.js

```diff
@@ -283,13 +283,14 @@ function cleanDeep(
  * @param {Object} configData
  * @todo Add description for the params
  */
-function validateForce(configData) {
-	if (configData.forging.force) {
-		var index = configData.constants.NETHASHES.indexOf(configData.nethash);
+function validateForce({ constants, forging, nethash }) {
+	if (forging.force) {
+		const { NETHASHES } = constants;
+		var index = NETHASHES.indexOf(nethash);
```

Comment:
Please use more meaningful variable name.

Reply:
I removed the variable since we didn't need it.